frontend/src/hocs/withScreencast/components/VideoRecordedDialog/style.scss

```diff
@@ -16,63 +16,56 @@
 
 @import "src/assets/css/theme";
 
-.streamlit-dialog.screencast {
-  width: 80%;
+.screencast-dialog {
+  display: grid;
+  grid-template-columns: 6em auto 6em;
 
-  .modal-body {
-    video {
-      width: 100%;
-      border-radius: $border-radius;
-    }
+  .first-column {
+    padding-right: 1rem;
+  }
 
-    .download-button {
-      margin-top: calc(-#{$border-width} - #{$input-btn-padding-y});
+  .second-step {
+    p {
+      margin: 0.5rem 0 0;
     }
+  }
 
-    .steps-container {
-      display: grid;
-
-      grid-template-columns: 6em auto 6em;
-
-      .first-column {
-        padding-right: 1rem;
-      }
+  .first-column,
+  .second-column,
+  .third-columnd {
```

Comment:
sorry missed this on the first pass but also it's been like this so not sure if fixing the typo will actually cause more grief or not... the component itself does not have the typo so i guess we've been living without some y padding 🤷 

```suggestion
  .third-column {
```